src/classes/RundownManager.ts

```diff
@@ -168,13 +169,22 @@ export class RundownManager {
 	/**
 	 * Adds a cue to the story. Returns the index of the newly added cue.
 	 */
-	private addCueToStory(story: INewsStory, cueKey: string): number {
-		story.cues.push([`${cueKey}=${story.fields.layout!.toUpperCase()}`])
+	private addCueToStory(story: INewsStory, cueKey: string, cueValue: string): number {
+		story.cues.push([`${cueKey}=${cueValue.toUpperCase()}`])
 		return story.cues.length - 1
 	}
 
 	private addDesignBgCueToStory(story: INewsStory): void {
-		const cueIndex = this.addCueToStory(story, 'DESIGN_BG')
+		const cueIndex = this.addCueToStory(story, 'DESIGN_BG', story.fields.layout!)
+		this.addLinkToStory(story, cueIndex)
+	}
+
+	public generateCueFromSchemaField(story: INewsStory): void {
```

Comment:
When I read generate, I expect it to return something. Suggestions:
- `updateStoryWithCueFromSchemaField`
- `updateStoryWithCueGeneratedFromSchemaField`
- `updateStoryWithSchemaFieldCue`
- `updateStoryWithSchemaFieldGeneratedCue`
- `insertCueFromSchemaFieldIntoStory`
- `insertCueGeneratedFromSchemaFieldIntoStory`
- `insertSchemaFieldCueIntoStory`
- `insertSchemaFieldGeneratedCueIntoStory`